src/server/engine_shard_set.cc

```diff
@@ -39,8 +49,260 @@ using absl::GetFlag;
 
 namespace {
 
+constexpr DbIndex DEFAULT_DB_INDEX = 0;
```

Comment:
Just using DB number 0 for now, so it would not be hard coded for now at other places